src/ui/MLFacet/MLFacetValues/MLFacetValues.ts

```diff
@@ -45,17 +45,21 @@ export class MLFacetValues {
   }
 
   public get selectedValues() {
-    return this.facetValues.filter(value => value.isSelected).map(value => value.value);
+    return this.facetValues.filter(value => value.isSelected).map(({ value }) => value);
   }
 
-  public get nonIdleValues() {
-    return this.facetValues.filter(value => !value.isIdle).map(value => value.value);
+  public get nonIdleFacetValues() {
+    return this.facetValues.filter(value => !value.isIdle);
   }
 
   public get hasSelectedValues() {
     return !!findWhere(this.facetValues, { state: FacetValueState.selected });
   }
 
+  public get hasNonIdleValues() {
```

Comment:
Could we change all the `nonIdle` wording to `active` instead ? So that we don't end up with "double negative" ?